csharp/ql/test/library-tests/utils/FilepathNormalizeTest.expected

```diff
@@ -0,0 +1,10 @@
+|  | . |
+| ./a/b/c/../d | a/b/d |
+| / | / |
+| /a/b/../c | /a/c |
+| /a/b/c/../../d/ | /a/d |
+| a/.. | . |
+| a/b/../c | a/c |
+| a/b//////c/./d/../e//d// | a/b/c/e/d |
+| a/b/c | a/b/c |
+| a/b/c/../../../../d/e/../f | ../d/f |
```

Comment:
Might be good to add a similar test but with an absolute path (e.g. `/a/b/c/../../../../d/e/../f` should resolve to `/d/f`.